tests/test_person_relations_service.py

```
@@ -0,0 +1,390 @@
+            {"source_type": "manual", "quote": "Quote 2"}, # new evidence
+        ],
+    )
+    assert action2 == "merged_into_existing"
+    assert rel2["relation_id"] == rel1["relation_id"]
+    assert "Initial note" in rel2["note"]
+    assert "Additional note" in rel2["note"]
+    assert len(rel2["evidence"]) == 2
+
+    # 6. Symmetric relation endpoint normalization
+    # peo_2 > peo_1, so subject should be normalized to peo_1, object to peo_2
+    rel_sym, action_sym = create_person_relation_in_tx(
```

Ruff (0.16.3)

[warning] Unpacked variable `action_sym` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)